features/steps/pdf_steps.py

```diff
@@ -24,21 +26,22 @@ def test_prefilled_pdf_for_applicant(context, applicant_name):
     context.test.assertTrue(sub.filled_pdfs.count() >= 1)
 
 
-@then('"{applicant_name}" should be in the new apps PDF for "{org_slug}"')
-def test_applicant_in_newapps_pdf_for_org(
-        context, applicant_name, org_slug):
-    first_name, last_name = applicant_name.split(' ')
-    sub = models.FormSubmission.objects.filter(
-        first_name=first_name, last_name=last_name).first()
-    new_apps_pdf = models.NewAppsPDF.objects.filter(
-        organization__slug=org_slug).first()
-    sub_ids = new_apps_pdf.applications.values_list(
-        'form_submission_id', flat=True)
-    context.test.assertIn(sub.id, sub_ids)
+@then('there should be a prebuilt PDF bundle for "{applicant_names}"')
+def test_applicant_in_prebuilt_pdf_for_org(context, applicant_names):
+    applicant_names = oxford_comma_text_to_list(applicant_names)
+    app_ids = []
+    for applicant_name in applicant_names:
+        first_name, last_name = applicant_name.split(' ')
+        app_ids.append(
+            models.Application.objects.filter(
+                form_submission__first_name=first_name,
+                form_submission__last_name=last_name).first().id)
+    prebuilt = pdf_service.get_prebuilt_pdf_bundle_for_app_id_set(app_ids)
+    context.test.assertTrue(prebuilt)
```

Comment:
The above step function checks for a prebuilt PDF _with a precise match_ for the given set of applicants. If one name is off from the result or the result is missing a name, this test step will fail.